receiver/azuremonitorreceiver/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+name: azuremonitorreceiver
```

Comment:
Please add a status section here, this will auto-generate the status table. See https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/7ab7c53d32806e3986f353eee7e659349ac693a6 for the changes